### ISSUE TITLE
Memed out Voice Player

### ISSUE THREAD

**Issue by Deivu** [Deivu/Kongou#1]
```js
Error: Already encoding
    at Piper.encode (/home/davexdme/kongou/node_modules/eris/lib/voice/Piper.js:43:32)
    at VoiceConnection.play (/home/davexdme/kongou/node_modules/eris/lib/voice/VoiceConnection.js:451:24)
    at Player.start (/home/davexdme/kongou/src/modules/moosikplayer.js:17:24)
    at Timeout.setTimeout [as _onTimeout] (/home/davexdme/kongou/src/modules/moosikplayer.js:28:10)
    at ontimeout (timers.js:424:11)
    at tryOnTimeout (timers.js:288:5)
    at listOnTimeout (timers.js:251:5)
    at Timer.processTimers (timers.js:211:10)
Error: Already encoding
    at Piper.encode (/home/davexdme/kongou/node_modules/eris/lib/voice/Piper.js:43:32)
    at VoiceConnection.play (/home/davexdme/kongou/node_modules/eris/lib/voice/VoiceConnection.js:451:24)
    at Player.start (/home/davexdme/kongou/src/modules/moosikplayer.js:17:24)
    at Timeout.setTimeout [as _onTimeout] (/home/davexdme/kongou/src/modules/moosikplayer.js:28:10)
    at ontimeout (timers.js:424:11)
    at tryOnTimeout (timers.js:288:5)
    at listOnTimeout (timers.js:251:5)
    at Timer.processTimers (timers.js:211:10)
Error: Already encoding
    at Piper.encode (/home/davexdme/kongou/node_modules/eris/lib/voice/Piper.js:43:32)
    at VoiceConnection.play (/home/davexdme/kongou/node_modules/eris/lib/voice/VoiceConnection.js:451:24)
    at Player.start (/home/davexdme/kongou/src/modules/moosikplayer.js:17:24)
    at Timeout.setTimeout [as _onTimeout] (/home/davexdme/kongou/src/modules/moosikplayer.js:28:10)
    at ontimeout (timers.js:424:11)
    at tryOnTimeout (timers.js:288:5)
    at listOnTimeout (timers.js:251:5)
    at Timer.processTimers (timers.js:211:10)
Error: Unable to encode source
    at VoiceConnection.play (/home/davexdme/kongou/node_modules/eris/lib/voice/VoiceConnection.js:452:32)
    at Player.start (/home/davexdme/kongou/src/modules/moosikplayer.js:17:24)
    at Timeout.setTimeout [as _onTimeout] (/home/davexdme/kongou/src/modules/moosikplayer.js:28:10)
    at ontimeout (timers.js:424:11)
    at tryOnTimeout (timers.js:288:5)
    at listOnTimeout (timers.js:251:5)
    at Timer.processTimers (timers.js:211:10)
Error: Unable to encode source
    at VoiceConnection.play (/home/davexdme/kongou/node_modules/eris/lib/voice/VoiceConnection.js:452:32)
    at Player.start (/home/davexdme/kongou/src/modules/moosikplayer.js:17:24)
    at Timeout.setTimeout [as _onTimeout] (/home/davexdme/kongou/src/modules/moosikplayer.js:28:10)
    at ontimeout (timers.js:424:11)
    at tryOnTimeout (timers.js:288:5)
    at listOnTimeout (timers.js:251:5)
    at Timer.processTimers (timers.js:211:10)
Error: Unable to encode source
    at VoiceConnection.play (/home/davexdme/kongou/node_modules/eris/lib/voice/VoiceConnection.js:452:32)
    at Player.start (/home/davexdme/kongou/src/modules/moosikplayer.js:17:24)
    at Timeout.setTimeout [as _onTimeout] (/home/davexdme/kongou/src/modules/moosikplayer.js:28:10)
    at ontimeout (timers.js:424:11)
    at tryOnTimeout (timers.js:288:5)
    at listOnTimeout (timers.js:251:5)
    at Timer.processTimers (timers.js:211:10)
Error: Already encoding
    at Piper.encode (/home/davexdme/kongou/node_modules/eris/lib/voice/Piper.js:43:32)
    at VoiceConnection.play (/home/davexdme/kongou/node_modules/eris/lib/voice/VoiceConnection.js:451:24)
    at Player.start (/home/davexdme/kongou/src/modules/moosikplayer.js:17:24)
    at Timeout.setTimeout [as _onTimeout] (/home/davexdme/kongou/src/modules/moosikplayer.js:28:10)
    at ontimeout (timers.js:424:11)
    at tryOnTimeout (timers.js:288:5)
    at listOnTimeout (timers.js:251:5)
    at Timer.processTimers (timers.js:211:10)
```
Error Stack. I cannot fix it, or more like I gave up fixing it. Any help will be appreciated.
`./src/modules/moosikplayer`

**Comment by Deivu** [comment]
it just plays the first 2 songs in queue then meme out.

**Comment by Deivu** [comment]
Fixed in Commit `793ac7c41d05b2143272cef4c1510311522b92ca`
